CMakeLists.txt

```diff
@@ -37,9 +38,9 @@ set(LONG_DESCRIPTION "testplugin Plugin is used to test out the ODraw API and de
 
 set(VERSION_MAJOR "1")
 set(VERSION_MINOR "0")
-set(VERSION_PATCH "132")
-set(VERSION_TWEAK "0")
-set(VERSION_DATE "01/08/2020")
+set(VERSION_PATCH "139")
```

Comment:
Please don't touch VERSION_PATCH, it belongs to upstream (Jon). If you have used a higher number 139 it's something in your fork, not related to the issue here which is the mingw metadata target.

Reply:
Ok, we had been working differently, but I can certainly do that.